_includes/event/judging.html

```diff
@@ -40,6 +40,22 @@ <h4>Usefulness</h4>
         </p>
       </div>
     </div>
-
+    {% if page.city == "Melbourne" %}
+    <div class="process">
+      <div class="row justify-content-center">
+        <div class="col-md-12">
+          <h3>New Judging Process</h3>
+          <p>
+            UNIHACK Melbourne is introducing a new judging process in 2018 to improve the
```

Comment:
The judging process is now being adopted by Sydney too. So can we change this to:
"UNIHACK is introducing a new judging process in 2018 to improve the..."

and we can get rid of the conditional city statement, and display on both cities